### PR TITLE
adding missing attributes to Models::Sequence

### PR DESCRIPTION
**Pull request by igorescobar**
Solving issues mentioned on #23

### CODE REVIEW

File: lib/invoicexpress/models/sequence.rb [hunk 1]
```diff
@@ -1,17 +1,35 @@
...
+      element :current_invoice_sequence_id, Integer
       element :current_invoice_number, Integer
+      element :current_credit_note_sequence_id, Integer
       element :current_credit_note_number, Integer
       element :current_debit_note_number, Integer
       element :default_sequence, Boolean
+      element :current_simplified_invoice_sequence_id, Integer
+      element :current_simplified_invoice_number, Integer
+      element :current_debit_note_sequence_id, Integer
+      element :current_debit_note_number, Integer
```
Comment (paulosilva86) [inline lib/invoicexpress/models/sequence.rb]: It seems like `:current_debit_note_number` was already defined above.

**Comment by igorescobar** [comment]
@paulosilva86 you are right! sorry about that!

**Comment by igorescobar** [comment]
done ;)

**Review by paulosilva86 — APPROVED** [review]

**Comment by paulosilva86** [comment]
@igorescobar a new version was published with this change: https://rubygems.org/gems/invoicexpress/versions/0.1.11

**Comment by igorescobar** [comment]
Thanks!

**Pull request merged**